modules/aviatrix-controller-initialize/variables.tf

```
@@ -0,0 +1,74 @@
+data "aws_caller_identity" "current" {}
+
+data "aws_region" "current" {}
+
+variable "controller_launch_wait_time" {
+  type        = number
+  description = "Controller launch wait time"
+  default     = 210
+}
+
+variable "aws_account_id" {
+  type        = string
+  description = "aws account id"
+}
+
+variable "public_ip" {
+  type        = string
+  description = "aviatrix controller public ip address(required)"
+}
+
+variable "private_ip" {
+  type        = string
+  description = "aviatrix controller private ip address(required)"
+}
+
+variable "admin_email" {
+  type        = string
+  description = "aviatrix controller admin email address"
+}
+
+variable "admin_password" {
+  type        = string
+  description = "aviatrix controller admin password"
+}
+
+variable "access_account_email" {
+  type        = string
+  description = "aviatrix controller access account email"
+}
+
+variable "controller_version" {
+  type        = string
+  default     = "latest"
+  description = "The version in which you want launch Aviatrix controller"
+}
+
+variable "access_account_name" {
+  type        = string
+  description = "Access account name"
+}
+
+variable "customer_license_id" {
+  type        = string
+  description = "aviatrix customer license id"
+  default     = ""
+}
+
+variable "ec2_role_name" {
+  type        = string
+  description = "EC2 role name"
+  default     = "aviatrix-role-ec2"
```

Not setting this variable to allow downstream logic to handle this differently (see  modules/aviatrix-controller-initialize/variables.tf line 71/72 suggestions)

```suggestion
  default     = ""
```